packages/api/src/beacon/routes/lightclient.ts

```diff
@@ -92,10 +106,15 @@ export function getReqSerializers(): ReqSerializers<Api, ReqTypes> {
 
 export function getReturnTypes(): ReturnTypes<Api> {
   return {
-    getUpdates: ContainerData(ArrayOf(ssz.altair.LightClientUpdate)),
-    getOptimisticUpdate: ContainerData(ssz.altair.LightClientOptimisticUpdate),
-    getFinalityUpdate: ContainerData(ssz.altair.LightClientFinalityUpdate),
-    getBootstrap: ContainerData(ssz.altair.LightClientBootstrap),
+    getUpdates: ArrayOf(
+      new ContainerType({
+        version: new StringType<ForkName>(),
+        data: ssz.altair.LightClientUpdate,
+      })
```

Comment:
WithVersion should work here too

Reply:
how would you use WithVersion here? This is like ArrayOf(WithVersion()) except it doesn't seem that logic works.

Reply:
> WithVersion should work here too

@dapplion I tried this but the types do not align. 

Is there anything specifically wrong with the current implementation making it a blocker? if so, can you maybe share how you expect `WithVersion` to be used to do the job better?

Reply:
Oh just checked the source, you are right Type vs TypeJson thing is awkward so it won't work here